### PR TITLE
Added tags to notifications, used action_id as id for action notifica…

### PR DESCRIPTION
**Pull request by Revenaunt**
…tions

### CODE REVIEW

File: src/main/java/org/tndata/android/compass/service/GcmIntentService.java [hunk 1]
```diff
@@ -106,12 +111,12 @@ private void sendNotification(String id, String msg, String title, String object
...
 
-        int notificationId = (int)(Math.random()*1000000);
         if (object_type.equals(Constants.ACTION_TYPE)){
-            try {
+            try{
                 Intent intent = new Intent(getApplicationContext(), ActionActivity.class);
                 intent.putExtra(ActionActivity.ACTION_ID_KEY, Integer.valueOf(object_id));
-                //intent.addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP | Intent.FLAG_ACTIVITY_CLEAR_TOP);
+
+                int notificationId = Integer.valueOf(object_id);
```
Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/service/GcmIntentService.java]: Isn't `object_id` null for the Behavior notifications, and if so, won't this cause a problem?

Comment (Revenaunt) [inline src/main/java/org/tndata/android/compass/service/GcmIntentService.java]: It is (I think), and it won't, that particular statement is enclosed in an action_type notification conditional.

Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/service/GcmIntentService.java]: Oh, you are correct. Cool.

**Comment by bradmontgomery** [comment]
:+1:

**Pull request merged**